frontend/src/components/graduations/GraduationLinkCardGrid.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import { GraduationLinkCard } from '@/components/graduations/GraduationLinkCard';
+import { Graduation } from '@/database/graduation';
+import { Grid } from '@mui/material';
+
+interface GraduationLinkCardGridProps {
+    graduations: Graduation[];
+}
+
+export default function GraduationLinkCardGrid({
+    graduations,
+}: GraduationLinkCardGridProps) {
+    graduations.sort(
+        (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
+    );
+
+    return (
+        <Grid container p='16px' spacing={2} alignItems='center' justifyContent='center'>
+            {graduations.map((grad) => (
+                <Grid
+                    key={`${grad.username}//${grad.createdAt}`}
```

Comment:
Bit of a nitpick, but could you remove the extra slash in the key?

Reply:
Done.